### ISSUE TITLE
ProtocolUnknownError on any URL

### ISSUE THREAD

**Issue by maaaks** [wkhtmltopdf/wkhtmltopdf#3371]
For example, if a page contains this...

```html
<link rel="stylesheet" href="http://localhost:5555/static/css/print/bill.print.css"/>
```

...I will get `ProtocolUnknownError`. Seems like it is triggered by `:` in the URL.

**Comment by maaaks** [comment]
Seems like it has some problems with _any_ URLs on the page. It renders the page only if I remove all `<link>` and `<img>` tags from it.

**Comment by maaaks** [comment]
Seems to be my problem. I have different errors with any PDF generation library I used because I tried to convert a HTML view into a PDF while processing an HTTP request, and since I use Flask which is not multi-threaded, it could not process other requests (for styles and images) until I finish. Thus, all libraries raise timeout errors and so on.

In other words, my problem most probably can be fixed by moving static files routing to the nginx level.

(I say «most probably» because I currently use other library, but I'm pretty sure that this was the only problem which I saw on all of them, including wkhtmltopdf.)
